lua/entities/sent_spawnpoint/init.lua

```diff
@@ -128,37 +129,39 @@ function ENT:Initialize()
     self.Entity.LinkedPlayers = {}
 
     local phys = self.Entity:GetPhysicsObject()
-    if ( phys:IsValid() ) then
-        phys:Wake()
-        phys:EnableDrag(true)
-        phys:EnableMotion(false)
-    end
+    if not phys:IsValid() then return end
+    
+    phys:Wake()
+    phys:EnableDrag( true )
+    phys:EnableMotion( false )
 end
 
 function ENT:OnRemove()
     local effectdata1 = EffectData()
     effectdata1:SetOrigin( self.Entity:GetPos() )
     util.Effect( "spawnpoint_start", effectdata1, true, true )
 
-    unlinkAllPlayersFromSpawnPoint(self.Entity, {})
+    unlinkAllPlayersFromSpawnPoint( self.Entity, {} )
 end
 
 function ENT:Use( player, caller )
-    if ( player.LinkedSpawnPoint and player.LinkedSpawnPoint == self.Entity ) then
+    local msg = ""
+    if player.LinkedSpawnPoint and player.LinkedSpawnPoint == self.Entity then
```

Comment:
Is this necessary?

Can we just do:

```lua
if player.LinkedSpawnPoint == self.Entity then
```

We don't need to navigate safely, this will just return `nil` if it doesn't exist.